### ISSUE TITLE
Gnome Classic Places remove

### ISSUE THREAD

**Issue by axet** [home-sweet-gnome/dash-to-panel#1014]
**Describe the idea or feature you'd like**

Gnome Classic mode add Places menu to the top bar, here is no way to remove or hide it.

**Is your feature request related to a problem? Please describe.**

Option to hide Places.

**Comment by jderose9** [comment]
Hi @axet! Not sure what distro you are on, but generally you want to go into GNOME Tweaks to the Extensions tab and disable the Places Indicator.

That said, and I've argued against this on the gnome developer distribution list about this in the past, but essentially when a "session" is active such as Gnome Classic mode, you are locked into a specific set of extensions being enabled by the distro. So, I'm going to assume your ability to disable that extension will be disabled.

My recommendation is to just log out and use the regular GNOME session rather than Classic Mode. Is there some feature you are getting from Classic Mode w/ dash-to-panel that you don't get from the regular GNOME mode?

**Comment by axet** [comment]
Hi! I'm using Debian 10. And you probably right, nothing let me to disable Places extension not even visible from Gnome Tweaks. (Also after hacking with different options and dconf I got Applications back, and can't get rid on it anymore) 

I'm trying to reduce memory footprint from Gnome by running some lightweight window manager, I tried Gnome Classic for that purpose and seeing annoying Applications and Places on my screen and bottom bar below. My notebook has 8GB or memory 2 of which dedicated to the video card. 6GB not much for these days.

I guess that is wrong place to ask. Thank you for detailed response!
